ci/fireci/fireci/gradle.py

```diff
@@ -30,7 +30,7 @@ def P(name, value):
 
 
 @stats.measure_call('gradle')
-def run(*args, gradle_opts='', workdir=None):
+def run(*args, gradle_opts='', workdir=None, gradle_bin_dir='.', check=True):
```

Comment:
Instead of changing workdir it might be simpler to run `./gradlew -p api-size ...`, wdyt?

Reply:
Done. I didn't even know that such a gradle flag exists. Thanks for pointing it out!